api/src/test/java/gov/cms/ab2d/api/controller/v2/StatusAPIV2Test.java

```diff
@@ -0,0 +1,112 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.util.List;
+
+import static gov.cms.ab2d.common.model.Role.SPONSOR_ROLE;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(classes = SpringBootApp.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+@Testcontainers
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_CLASS)
+class StatusAPIV2Test {
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @Autowired
+  JobClientMock jobClientMock;
+
+  @Container
+  private static final PostgreSQLContainer postgreSQLContainer = new AB2DPostgresqlContainer();
+
+  @Container
+  private static final AB2DLocalstackContainer localstackContainer = new AB2DLocalstackContainer();
+
+  @Autowired
+  private TestUtil testUtil;
+
+  @Autowired
+  private DataSetup dataSetup;
+
+  @Autowired
+  AmazonSQS amazonSQS;
+
+  @Autowired
+  SQSEventClient sqsEventClient;
+
+  private String token;
+
+  @BeforeEach
+  public void setup() throws Exception {
+    token = testUtil.setupToken(List.of(SPONSOR_ROLE));
+    testUtil.turnMaintenanceModeOff();
+  }
+
+  @AfterEach
+  public void cleanup() {
+    dataSetup.cleanup();
+    jobClientMock.cleanupAll();
+  }
+
+  @Test
+  void testStatus() throws Exception {
+    // start a job
+    this.mockMvc.perform(
+      get("https://localhost:8443/api/v2/fhir/Patient/$export").contentType(MediaType.APPLICATION_JSON)
+              .header("Authorization", "Bearer " + token)
+              .header("X-Forwarded-Proto", "https"));
+
+    // get the job status
+    String jobUuid = jobClientMock.pickAJob();
+    ResultActions resultActions = this.mockMvc.perform(
+      get(String.format("https://localhost:8443/api/v2/fhir/Job/%s/$status", jobUuid)).contentType(MediaType.APPLICATION_JSON)
+        .header("Authorization", "Bearer " + token));
+
+    // assert ok
+    resultActions.andExpect(status().isOk());
+  }
+
+  @Test
+  void testDelete() throws Exception {
+    // start a job
+    this.mockMvc.perform(
+      get("https://localhost:8443/api/v2/fhir/Patient/$export").contentType(MediaType.APPLICATION_JSON)
```

Comment:
Same as above regarding the content type :)

Reply:
Updated ✅ https://github.com/CMSgov/ab2d/pull/1381/commits/c42c844eb2de213b34628299bf67cb0095b1dc3c